uavcan/equipment/power/1093.ECB.uavcan

```diff
@@ -0,0 +1,39 @@
+#
+# Electronic Circuit Breaker Device
+#
+# Typical publishing rate should be around 1 Hz.
+#
+
+#
+# ECB technical charateristics
+#
+uint4 number_of_input_channels  # Total number of power supplies
+uint4 number_of_output_channels # Total number of switched power circuits
+uint8 nominal_voltage           # Nominal voltage of ECB (for example 14 or 28 VDC)
+uint8 maximum_total_current     # Maximum total current that ECB is capable withstand
```

Comment:
I propose to make voltage and current of type `float16` for reasons of consistency. The added overhead is negligible.